### PR TITLE
Update manage-addins-in-the-admin-center.md

### PR DESCRIPTION
**Pull request by msbemba**
added a note that preventing users from accessing the office store will also affect them from Sideloading Office Add-ins for testing

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3121

File: microsoft-365/admin/manage/manage-addins-in-the-admin-center.md [hunk 1]
```diff
@@ -129,6 +129,9 @@ This does not prevent an administrator from using Centralized Deployment to assi
   
 To prevent a user from signing in with a Microsoft account, you can restrict logon to use only the organizational account. For more information, see [Identity, authentication, and authorization in Office 2016](https://technet.microsoft.com/library/jj683102%28v=office.16%29.aspx).  
 
+ > [!NOTE]
+ >  Preventing users from accessing the office store will also affect them from [Sideloading Office Add-ins for testing](https://docs.microsoft.com/office/dev/add-ins/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins)
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-addins-in-the-admin-center.md]:
```suggestion
> Preventing users from accessing the office store will also prevent them from [Sideloading Office Add-ins for testing](https://docs.microsoft.com/office/dev/add-ins/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins).
```

File: microsoft-365/admin/manage/manage-addins-in-the-admin-center.md [hunk 2]
```diff
@@ -129,6 +129,9 @@ This does not prevent an administrator from using Centralized Deployment to assi
   
 To prevent a user from signing in with a Microsoft account, you can restrict logon to use only the organizational account. For more information, see [Identity, authentication, and authorization in Office 2016](https://technet.microsoft.com/library/jj683102%28v=office.16%29.aspx).  
 
+ > [!NOTE]
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/manage-addins-in-the-admin-center.md]:
```suggestion
> [!NOTE]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9 Please approve the changes. Thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks.

**Pull request merged**